### PR TITLE
bump toml from ^0.5 to ^0.7.6

### PR DESCRIPTION
**Pull request by PratikMahajan**
update dependencies

### CODE REVIEW

File: graph-builder/src/config/file.rs [hunk 1]
```diff
@@ -41,7 +41,7 @@ impl FileOptions {
 
         let mut content = vec![];
         bufrd.read_to_end(&mut content)?;
-        let cfg = toml::from_slice(&content).context(format!(
+        let cfg = toml::from_str(std::str::from_utf8(&content)?).context(format!(
```
Comment (LalatenduMohanty) [inline graph-builder/src/config/file.rs:44]: @PratikMahajan Can you please add why these change are required because of the toml version change?

**Review by LalatenduMohanty — CHANGES_REQUESTED** [review]

File: graph-builder/src/config/file.rs (hunk 1 above)
Comment (PratikMahajan) [reply graph-builder/src/config/file.rs:44]: dependency bump was a breaking change and required a corresponding code change as well

**Review by petr-muller — APPROVED** [review]
